backend/src/test/java/gov/cdc/usds/simplereport/api/graphql/ApiUserManagementTest.java

```diff
@@ -816,31 +819,22 @@ void updateUserPrivileges_orgAdmin_success() {
     assertUserCanAccessExactFacilities(
         updateUser, Set.of(TestUserIdentities.TEST_FACILITY_1, TestUserIdentities.TEST_FACILITY_2));
     assertUserCanAccessAllFacilities(updateUser);
+  }
 
-    // Update 4: USER, access to no facilities
-    updatePrivilegesVariables =
-        getUpdateUserPrivilegesVariables(
-            id, Role.USER, false, extractFacilityUuidsFromUser(updateUser, Set.of()));
```

Comment:
no longer expect success here so I moved this test case into the following test and confirm failure